clients/python-wrapper/utests/test_client.py

```diff
@@ -5,19 +5,19 @@
 
 from lakefs import config as client_config
 
-test_server = "https://test_server/api/v1"
-test_access_key_id = "test_access_key_id"
-test_secret_access_key = "test_secret_access_key"
-test_config = f'''
+TEST_SERVER = "https://test_server/api/v1"
```

Comment:
It supports it - but the thing is it attaches it to the host if not exists and then whenever we validate expected against actual configuration we need to explicitly add it.
This pertains to the next comment as well.
Since we are not testing the SDK package - I don't think it hurts to leave it, WDYT?

Reply:
We do support it, but we don't guide the uses to use it. I think we should test the way we document how we like to use it (without end path).